base/env.jl

```diff
@@ -98,6 +98,36 @@ Base.EnvDict with "N" entries:
 """
 const ENV = EnvDict()
 
+"""
+    Base.get_bool_env(name::String, default::Bool = false)::Bool
+
+Evaluate whether the value of environnment variable `name` is truthy or falsy,
+and error if none are recognized. If the variable is not set, or is set to "",
+return `default`.
+
+Recognized values are:
+    truthy: "t", "true", "y", "yes", "1"
+    falsy:  "f", "false", "n", "no", "0"
+"""
+function get_bool_env(name::String, default::Bool = false)
+    haskey(ENV, name) || return default
+    val = lowercase(get(ENV, name, ""))
+    if isempty(val)
+        return default
+    elseif val in ("t", "true", "y", "yes", "1")
+        return true
+    elseif val in ("f", "false", "n", "no", "0")
+        return false
+    else
+        error("""
+        Failed to parse the value `$val` of environment variable `$name` as Boolean.
+        Recognized values are:
+            truthy: "t", "true", "y", "yes", "1"
+            falsy:  "f", "false", "n", "no", "0"
+        """)
```

Comment:
I'd do something like
```suggestion
    truthy = ("t", "true", "y", "yes", "1")
    falsy = ("f", "false", "n", "no", "0")
    if isempty(val)
        return default
    elseif val in truthy
        return true
    elseif val in falsy
        return false
    else
        error("""
        Failed to parse the value `$val` of environment variable `$name` as Boolean.
        Recognized values are:
            truthy: $(join(repr.(truthy), ", "))
            falsy:  $(join(repr.(falsy), ", "))
        """)
```

Reply:
Sorry, yes. I'll co-author you. Meant to do that first time around